src/runtime/virtcontainers/container.go

```diff
@@ -656,6 +658,8 @@ func (c *Container) createBlockDevices(ctx context.Context) error {
 						continue
 					}
 					c.mounts[i].FSGroupChangePolicy = volume.FSGroupChangePolicy(value)
+				case sensitiveMountOptions:
```

Comment:
other case exercise some form of validation - we are sure we don't need this for the value this case?

Reply:
can you please elaborate on the ask?

Reply:
Other than checking that the option string is properly formatted, we can't do much validation since the driver could pass various options depending on what the user specifies.